_posts/0000-01-02-pawan-giri.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
*Intelligence is a very valuable thing innit, my friend?
And usually, it comes far too fucking late.*